apps/aecore/lib/aecore/structures/tx_data.ex

```diff
@@ -15,14 +15,19 @@ defmodule Aecore.Structures.TxData do
      - to_acc: To account is the public address of the account receiving the transaction
      - value: The amount of a transaction
   """
-  defstruct nonce: 0,
-            from_acc: "",
-            to_acc: "",
-            value: 10
+  defstruct [:nonce,
+             :from_acc,
+             :to_acc,
+             :value]
   use ExConstructor
 
-  @spec create() :: tx_data()
-  def create do
-    TxData.new(%{})
+  @spec create(map()) :: {:ok, %TxData{}}
+  def create(%{from_acc: from_pubkey, to_acc: to_pubkey, value: value}) do
```

Comment:
Why do we need to pass a map to this function? Seems better to do it like in Header.create()

Reply:
Yes you are correct, I'll change it